python/stlib/physics/mixedmaterial/rigidification.py

```diff
@@ -15,140 +15,147 @@
 *******************************
 .. autofunction:: Rigidify
 
-    
-Contributors: 
+
+Contributors:
         damien.marchal@univ-lille.fr
         eulalie.coevoet@inria.fr
 """
 
 
-from splib.numerics import Vec3, Quat, sdiv, RigidDof, getOrientedBoxFromTransform 
+from splib.numerics import Vec3, Quat, sdiv
+
 
-def Rigidify(targetObject, sourceObject, frameOrientation, groupIndices, name=None):
+def Rigidify(targetObject, sourceObject, frameOrientation, groupIndices, framePosition=None, name=None):
```

Comment:
Why not having a single attribute frames=
[[0,0,0]] <= eulerAngales
[[0,0,0,0]] <= quaternion
[[0,0,0,0,0,0] <= position+eulerAngles
[[0,0,0,0,0,0, 1] <= posittion+quaternion

Reply:
Good idea